graphql/schema.graphql

```diff
@@ -124,10 +124,66 @@ type Build {
   name: String!
 
   "Start time."
-  starttime: DateTime!
+  startTime: DateTime! @rename(attribute: "starttime")
 
   "End time."
-  endtime: DateTime!
+  endTime: DateTime! @rename(attribute: "endtime")
+
+  "Submission time."
+  submissionTime: DateTime! @rename(attribute: "submittime")
+
+  "The buildstamp generted by CTest. Example: 20091218-1414-Experimental"
+  stamp: String!
+
+  # TODO: Create a designated UUID GraphQL type
+  "UUID."
+  uuid: String!
+
+  # TODO: Determine if there are more specific values we can use for this
+  "Type."
+  buildType: String! @rename(attribute: "type")
+
+  "The version of CTest used to generate this build. Example: ctest-2.9.20091218"
+  generator: String!
+
+  "The command used to drive the build. Example: /usr/local/bin/cmake --build . --config Release
+  command: String!
+
+  "Log."
+  log: String!
```

Comment:
I didn't spend a ton of time on this, but I couldn't find any meaningful data for this field. Maybe we should leave it out of the graphql schema for now? I'm worried users will expect to be able to retrieve the full build log from this field, which CTest/CDash do not necessarily provide.

Reply:
Seems reasonable to me.  I'll remove it from the schema.  I don't see any member variables in the project class, so it's possible that this is now a dead column.